packages/strapi-design-system/src/EmptyStateLayout/EmptyStateLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+
+import { Box, BoxProps } from '../Box';
+import { Flex, FlexProps } from '../Flex';
+import { Typography } from '../Typography';
+
+export interface EmptyStateLayoutProps extends FlexProps {
+  action?: React.ReactNode;
+  content: React.ReactNode;
+  icon: React.ReactNode;
+}
+
+const EmptyStateIconWrapper = styled(Box)<BoxProps>`
```

Comment:
```suggestion
const EmptyStateIconWrapper = styled(Box)`
```

You don't need to add this :)

Reply:
Ok. I thought it might make sense, so that the next developer doesn't have to type it :)

Reply:
It's inferred automatically, the only time you need to write them is if you're passing props that `this` styled-component  uses as opposed to `Box` e.g. you can use the value `background` in this styled component because it's part of the props of `Box`, but if you passed `hasBackground` you'd need to type that type, but you don't need to extend because `styled-components` creates the union... Maybe a diagram would help if that doesn't make sense? 👀

Reply:
It makes sense, thanks. TS seems to be way more intelligent than flow-type where you had to type every function interface.